libraries/AP_GPS/AP_GPS_NOVA.cpp

```diff
@@ -226,7 +226,7 @@ AP_GPS_NOVA::process_message(void)
                 case 34: // narrow float
                 case 48: // l1 int
                 case 50: // narrow int
-                    state.status = AP_GPS::GPS_OK_FIX_3D_RTK;
+                    state.status = AP_GPS::GPS_OK_FIX_3D_RTK_FLOAT;
```

Comment:
> and I considered the number of options here unclear

What do you mean? I've looked at the documentation and the comments here are basically the same, so I would say that indeed *int* means fixed RTK.